core/flow/rule.go

```diff
@@ -34,16 +45,62 @@ const (
 	AssociatedResource
 )
 
-// ControlBehavior indicates the traffic shaping behaviour.
-type ControlBehavior int32
+func (s RelationStrategy) String() string {
+	switch s {
+	case Direct:
+		return "Direct"
+	case AssociatedResource:
+		return "AssociatedResource"
+	default:
+		return "Undefined"
+	}
+}
+
+type AllocationStrategy int32
 
 const (
-	Reject ControlBehavior = iota
+	Default AllocationStrategy = iota
```

Comment:
"Default" seems too generic?

Reply:
Maybe Direct?

Reply:
The naming had been updated. 
How do you think of?